src/test/java/org/opensrp/web/rest/ClientFormResourceTest.java

```diff
@@ -497,39 +677,38 @@ public void testAddClientFormWhenGivenInvalidYamlShouldReturn400() throws Except
     @Test
     public void testAddClientFormWhenGivenPropertiesFile() throws Exception {
         String formIdentifier = "opd/opd_register.properties";
-        String formVersion = "0.1.1";
         String formName = "Registration properties file";
 
         MockMultipartFile file = new MockMultipartFile("form", "path/to/opd/opd_register.properties",
                 "application/octet-stream", TestFileContent.JMAG_PROPERTIES_FILE_CONTENT.getBytes());
 
+        when(manifestService.getAllManifest(anyInt())).thenReturn(getManifestList());
         when(clientFormService.addClientForm(any(ClientForm.class), any(ClientFormMetadata.class))).thenReturn(mock(ClientFormService.CompleteClientForm.class));
 
         mockMvc.perform(
                 fileUpload(BASE_URL)
                         .file(file)
                         .param("form_identifier", formIdentifier)
-                        .param("form_version", formVersion)
                         .param("form_name", formName))
                 .andExpect(status().isCreated())
                 .andReturn();
 
         ArgumentCaptor<ClientForm> clientFormArgumentCaptor = ArgumentCaptor.forClass(ClientForm.class);
         ArgumentCaptor<ClientFormMetadata> clientFormMetadataArgumentCaptor = ArgumentCaptor.forClass(ClientFormMetadata.class);
         verify(clientFormService).addClientForm(clientFormArgumentCaptor.capture(), clientFormMetadataArgumentCaptor.capture());
+        verify(manifestService).getAllManifest(anyInt());
 
         assertEquals(TestFileContent.JMAG_PROPERTIES_FILE_CONTENT, clientFormArgumentCaptor.getValue().getJson().toString());
         ClientFormMetadata clientFormMetadata = clientFormMetadataArgumentCaptor.getValue();
         assertEquals(formIdentifier, clientFormMetadata.getIdentifier());
-        assertEquals(formVersion, clientFormMetadata.getVersion());
+        assertEquals("0.0.2", clientFormMetadata.getVersion());
         assertEquals(formName, clientFormMetadata.getLabel());
         assertNull(clientFormMetadata.getModule());
     }
 
     @Test
     public void testAddClientFormWhenGivenInvalidPropertiesFileShouldReturn400() throws Exception {
         String formIdentifier = "opd/opd_register.properties";
-        String formVersion = "0.1.1";
```

Comment:
This were removed since my understanding was. We aren't making the form version optional but removing the ability for a user/dev to actually set one.  In that case we don't need to pass in a form version.  

We can make in possible for devs to specify the form version but the our UI should restrict the users from doing it.